VideoLocker/res/layout/fragment_course_dashboard.xml

```diff
@@ -18,28 +18,31 @@
                 android:id="@+id/header_image_view"
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
-                android:scaleType="fitXY"
                 android:contentDescription="@null"
+                android:scaleType="fitXY"
                 tools:src="@drawable/edx_map_login" />
 
             <LinearLayout
                 android:layout_width="match_parent"
-                android:layout_height="70dp"
+                android:layout_height="wrap_content"
                 android:layout_gravity="bottom"
                 android:background="@color/transparent_white_85"
+                android:minHeight="@dimen/course_detail_card_height"
                 android:orientation="horizontal">
 
                 <LinearLayout
                     android:layout_width="0dp"
-                    android:layout_height="match_parent"
+                    android:layout_height="wrap_content"
                     android:layout_gravity="bottom"
                     android:layout_weight="1"
                     android:orientation="vertical"
-                    android:padding="10dp">
+                    android:padding="@dimen/widget_margin">
 
                     <TextView
                         android:id="@+id/course_detail_name"
                         style="@style/course_card_title"
+                        android:ellipsize="none"
+                        android:singleLine="false"
```

Comment:
I have added a query for Liz on the ticket regarding whether we should truly allow the title to expand indefinitely; let's wait on her response before merging this.